app/assets/stylesheets/middleware_topology.css

```diff
@@ -0,0 +1,4 @@
+.middleware .kube-topology g.MiddlewareDatasource text.glyph {
```

Comment:
you need to add this file dependency to https://github.com/ManageIQ/manageiq/blob/master/app/assets/stylesheets/application.css